tests/test_emitter.py

```diff
@@ -105,6 +103,25 @@ def test_create():
         assert isinstance(event, DirModifiedEvent)
 
 
+@pytest.mark.flaky(max_runs=5, min_passes=1, rerun_filter=rerun_filter)
+@pytest.mark.skipif(
+    platform.is_darwin() or platform.is_windows(),
+    reason="Windows and macOS enforce proper encoding"
+)
+def test_create_wrong_encoding():
+    start_watching()
+    open(p('a_\udce4'), 'a').close()
+
+    event = event_queue.get(timeout=5)[0]
+    assert event.src_path == p('a_\udce4')
+    assert isinstance(event, FileCreatedEvent)
+
+    if not platform.is_windows():
```

Comment:
Not needed as the test is never ran on Windows.

Reply:
Well I merge right now and will do the clean-up before the release.

Reply:
Ah, yes, I copy-pasted this without paying attention. Thanks for flagging!